.mailmap

```diff
@@ -74,6 +74,7 @@ Clement Lebrun <clement.lebrun.31@gmail.com> <clem_noob@hotmail.fr>
 Colin P. McCabe <colinm@hq.newdream.net> <cmccabe@alumni.cmu.edu>
 Colin P. McCabe <colinm@hq.newdream.net> <cmccabe@fatty.ops.newdream.net>
 Colin Walters <walters@redhat.com> <walters@verbum.org>
+chenliuzhong <liuzhong.chen@easystack.cn>
```

Comment:
It looks like this is a github username `chenliuzhong`. Please provide your actual name here in the alphabetical order. 

@tchaikov Could you please suggest if this entry is necessary as `chenliuzhong` is having only one Email ID ? Thanks.

Reply:
Yeah keeping an entry in mailmap always helps when there are future mistakes etc.

Reply:
@theanalyst oh okay, I have seen many people with single emails in .mailmap.